packages/snaps-execution-environments/src/iframe/index.ts

```diff
@@ -1,7 +1,12 @@
+// eslint-disable-next-line import/no-unassigned-import
+import 'ses';
```

Comment:
this should not be imported into the bundle after we add the webpack plugin.

Reply:
Indeed it should not, we should use the LM Compartment

Reply:
Yeah, I added it temporarily so that the execution environment works. Without it, `Compartment` would be undefined.